### PR TITLE
Issue #80: Implementing sending of emails on membership success

### LINKED ISSUE #80: Send emails for membership succeed pages (`send_membership_membership_success_message`?)

### PR DESCRIPTION
**Pull request by lilliputten**
Send email for membership succeed: added email message templates (`membership_registration_new_success_message_*`). Minor refactors: extracted some helper methods (`get_membership_cost`, `check_for_available_membership`), removed unused code.

### CODE REVIEW

File: dds_registration/models.py [hunk 1]
```diff
@@ -84,6 +84,10 @@ def sync_email_and_username(self):
             self._original_username = self.username
             # TODO: To do smth else if email has changed?
 
+    def get_full_name_or_email(self):
```
Comment (cmutel) [inline dds_registration/models.py:87]: I don't think this is right - we have to know the names of people for billing them, addressing them, etc. So I don't think we can allow people to register without providing their names.

File: dds_registration/templates/dds_registration/membership_registration_new_success_message_subject.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{# ex: set ft=htmldjango : #}
+
+[DdS]: You almost have become a DdS Member
```
Comment (cmutel) [inline dds_registration/templates/dds_registration/membership_registration_new_success_message_subject.txt:3]: We can delete this - I will polish the language later.

File: dds_registration/models.py (hunk 1 above)
Comment (lilliputten) [reply dds_registration/models.py:87]: At the moment these values sometimes optional. Probably we have to make one more issue to fix that.

Comment (lilliputten) [reply dds_registration/models.py:87]: Added #90 ("Ensure that user name (last and first) are required").

**Pull request merged**